### PR TITLE
OSDOCS 10255:Updated Release notes for 1.13.1

### PR DESCRIPTION
**Pull request by subhtk**
Version(s): cert-manager- 1.13.1, enterprise-4.16

Issue:  [OSDOCS-10255](https://issues.redhat.com/browse/OSDOCS-10255)

Link to docs preview: [Docs Preview](https://74930--ocpdocs-pr.netlify.app/openshift-enterprise/latest/security/cert_manager_operator/cert-manager-operator-release-notes.html#cert-manager-operator-release-notes-1-13)

SME review:
- [x] SME has approved this change.

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: security/cert_manager_operator/cert-manager-operator-release-notes.adoc [hunk 1]
```diff
@@ -13,31 +13,23 @@ These release notes track the development of {cert-manager-operator}.
...
-Issued: 2024-01-16
+Issued: 
 
-The following advisory is available for the {cert-manager-operator} 1.13.0:
+The following advisory is available for the {cert-manager-operator} 1.13.1:
 
 * link:https://access.redhat.com/errata/RHEA-2024:0259[RHEA-2024:0259]
 
-Version `1.13.0` of the {cert-manager-operator} is based on the upstream cert-manager version `v1.13.3`. For more information, see the link:https://cert-manager.io/docs/release-notes/release-notes-1.13/#v1133[cert-manager project release notes for v1.13.0].
+Version `1.13.1` of the {cert-manager-operator} is based on the upstream cert-manager version `v1.13.5`. For more information, see the link:https://cert-manager.io/docs/release-notes/release-notes-1.13/#v1133[cert-manager project release notes for v1.13.0].
```
Comment (lunarwhite) [inline security/cert_manager_operator/cert-manager-operator-release-notes.adoc:24]:
```suggestion
Version `1.13.1` of the {cert-manager-operator} is based on the upstream cert-manager version `v1.13.5`. For more information, see the link:https://cert-manager.io/docs/release-notes/release-notes-1.13/#v1135[cert-manager project release notes for v1.13.5].
```

**Comment by lunarwhite** [comment]
Is it intended to remove the 1.13.0 release notes? (Not quite familiar with how this repo's branching works)

**Review by swghosh — CHANGES_REQUESTED** [review]

File: security/cert_manager_operator/cert-manager-operator-release-notes.adoc (hunk 1 above)
Comment (subhtk) [reply security/cert_manager_operator/cert-manager-operator-release-notes.adoc:24]: Ack.

File: security/cert_manager_operator/cert-manager-operator-release-notes.adoc [hunk 2]
```diff
@@ -13,31 +13,23 @@ These release notes track the development of {cert-manager-operator}.
 For more information, see xref:../../security/cert_manager_operator/index.adoc#cert-manager-operator-about[About the {cert-manager-operator}].
 
 [id="cert-manager-operator-release-notes-1-13"]
-== {cert-manager-operator} 1.13.0
+== {cert-manager-operator} 1.13.1
```
[the remark this answers is not in the document]
Comment (subhtk) [reply security/cert_manager_operator/cert-manager-operator-release-notes.adoc:16]: Ack. I will update it accordingly.

File: security/cert_manager_operator/cert-manager-operator-release-notes.adoc [hunk 3]
```diff
@@ -13,31 +13,23 @@ These release notes track the development of {cert-manager-operator}.
...
 
 [id="cert-manager-operator-1-13-CVEs"]
 === CVEs
 
-* link:https://access.redhat.com/security/cve/CVE-2023-39615[CVE-2023-39615]
-* link:https://access.redhat.com/security/cve/CVE-2023-3978[CVE-2023-3978]
-* link:https://access.redhat.com/security/cve/CVE-2023-37788[CVE-2023-37788]
-* link:https://access.redhat.com/security/cve/CVE-2023-29406[CVE-2023-29406]
+* link:https://access.redhat.com/security/cve/CVE-2023-45288[CVE-2023-45288]
+
```
[the remark this answers is not in the document]
Comment (subhtk) [reply security/cert_manager_operator/cert-manager-operator-release-notes.adoc:35]: Done.

[the remark this answers is not in the document]
Comment (subhtk) [reply security/cert_manager_operator/cert-manager-operator-release-notes.adoc:35]: Ack.

[the remark this answers is not in the document]
Comment (subhtk) [reply security/cert_manager_operator/cert-manager-operator-release-notes.adoc:35]: Also, can you provide the errata link for the 1.13.1 version?

**Comment by snarayan-redhat** [comment]
@subhtk This should apply to 4.13+. Why is this targeting 4.16?

**Comment by subhtk** [comment]
According to jira- the release note update is only for 4.16, for now.

**Comment by subhtk** [comment]
Hi, yes, my apologies. We do need from 4.13 to 4.16, Let me create another PR with the commit.

**Comment by subhtk** [comment]
Created https://github.com/openshift/openshift-docs/pull/75962 to incorporate applicable versions.

**Comment by subhtk** [comment]
Closing the PR due to incorrect version - changes tracked through new PR - https://github.com/openshift/openshift-docs/pull/75962.

**Pull request closed without merging**